.github/workflows/release_packages.yml

```
@@ -0,0 +1,54 @@
+name: "Release packages"
+
+on:
+  workflow_dispatch:
+
+jobs:
+  release-packages:
+    runs-on: ubuntu-latest
+    
+    permissions:
+      contents: write
+      id-token: write
```

```suggestion
    
    permissions:
      contents: write
      id-token: write
```
This is only needed if using [`vault-action`](https://github.com/hashicorp/vault-action). It is used to get npm token from vault in publish-branch workflow